pkg/web_app/lib/src/autocompletion.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:html';
+
+final _widgets = {
+  'autocompletion': _createAutocompletionWidget,
```

Comment:
should this go in as a separate PR?

Reply:
yes, this thing is not done yet :see_no_evil: